### ISSUE TITLE
Multiple sites share an instance of riprap

### ISSUE THREAD

**Issue by kylehuynh205** [mjordan/riprap#78]
Hello, 

I setup riprap with: multiple sites sharing one instance of riprap and one riprap database.

However, the riprap report in each of those sites (at `admin/reports/islandora_riprap`) show failed events of all the sites because of the shared database, I'm wondering if there is a way to categorize riprap events for each site when sharing the database ?

Thank you.

**Comment by mjordan** [comment]
No, there isn't currently a way to do this. I will need to look more closely at all the impacted parts of Riprap, but I think adding a `repository` column to the db, having each repository use a unique ID in that column, and then making reports only show the entries for the specified repository/repositories would be a solution. But several parts of Riprap would need to be changed, for example the persister plugin, Islandora Riprap Integration, and maybe some others. Let me take a closer look.
